custom-ktlint-rules/src/main/resources/META-INF/services/com.pinterest.ktlint.core.RuleSetProviderV2

```diff
@@ -0,0 +1 @@
+com.appunite.loudius.rules.CustomRuleSetProvider
```

Comment:
**chore** No new line at the end of the file

Reply:
I think this file should have just one line as provided in the example file: https://github.com/pinterest/ktlint/blob/0.48.2/ktlint-ruleset-template/src/main/resources/META-INF/services/com.pinterest.ktlint.core.RuleSetProviderV2